helm_deploy/values-preprod.yaml

```diff
@@ -14,8 +14,8 @@ generic-service:
     API_BASE_URL_MAPPING: https://nomis-sync-prisoner-mapping-preprod.hmpps.service.justice.gov.uk
     API_BASE_URL_VISITS: https://visit-scheduler-preprod.prison.service.justice.gov.uk
     API_BASE_URL_INCENTIVES: https://incentives-api-preprod.hmpps.service.justice.gov.uk
-    API_BASE_URL_ACTIVITIES: https://activities-api-dev.prison.service.justice.gov.uk
     # point at self until real API exists
+    API_BASE_URL_ACTIVITIES: https://prisoner-to-nomis-update-preprod.hmpps.service.justice.gov.uk
```

Comment:
pointing at self seems a bit safer

Reply:
Does this mean we should turn off the feature switches in preprod?

Reply:
yes that makes sense 👍